src/tests/dbus-tests/test_50_block.py

```diff
@@ -90,7 +90,7 @@ def test_open(self):
         self.assertIsNotNone(dbus_fd)
 
         fd = dbus_fd.take()
-        mode = fcntl.fcntl(fd, fcntl.F_GETFL) & os.O_ACCMODE
+        mode = fcntl.fcntl(fd, fcntl.F_GETFL) & 3  # os.O_ACCMODE
```

Comment:
Would be nicer to try to import the constant and define it on our own if the import fails somewhere above.